test/runtime/Policies.go

```diff
@@ -1010,6 +1013,78 @@ var _ = Describe("RuntimeValidatedPolicies", func() {
 
 		})
 	})
+	Context("DROP_ALL  Policy test", func() {
+		AfterEach(func() {
+			vm.ContainerRm(dropAllContainer).ExpectSuccess("Container dropAllContainer cannot be deleted")
+		})
+
+		It("DROP_ALL Ingress Policy test", func() {
+			res := vm.ContainerCreate(dropAllContainer, "tgraf/netperf", helpers.CiliumDockerNetwork, "", "")
```

Comment:
could you change `tgraf/netperf` for `helpers.NetperfImage` ?

Reply:
done